Upstream: No
Reason: no need to require programs during the build
Source: Daniel Mierswa <impulze@impulze.org>
Updated for 2.11.05: Timo Gurr <tgurr@exherbo.org>

diff -Naur nasm-2.11.05.orig/configure.in nasm-2.11.05/configure.in
--- nasm-2.11.05.orig/configure.in	2014-05-21 17:44:06.000000000 +0200
+++ nasm-2.11.05/configure.in	2014-10-09 18:39:12.460044012 +0200
@@ -67,12 +67,18 @@
 PA_ADD_CFLAGS([-pedantic])
 
 dnl Look for programs...
-AC_CHECK_PROGS(NROFF,    nroff,    false)
-AC_CHECK_PROGS(ASCIIDOC, asciidoc, false)
-AC_CHECK_PROGS(XMLTO,    xmlto,    false)
-AC_CHECK_PROGS(ACRODIST, acrodist, false)
-AC_CHECK_PROGS(PS2PDF,   ps2pdf,   false)
-AC_CHECK_PROGS(PSTOPDF,  pstopdf,  false)
+AC_ARG_WITH([nroff], [AS_HELP_STRING([--with-nroff], [use nroff])])
+test "$with_nroff" != no && AC_CHECK_PROGS(NROFF,    nroff,    echo)
+AC_ARG_WITH([asciidoc], [AS_HELP_STRING([--with-asciidoc], [use asciidoc])])
+test "$with_asciidoc" != no && AC_CHECK_PROGS(ASCIIDOC, asciidoc, false)
+AC_ARG_WITH([xmlto], [AS_HELP_STRING([--with-xmlto], [use xmlto])])
+test "$with_xmlto" != no && AC_CHECK_PROGS(XMLTO, xmlto, false)
+AC_ARG_WITH([acrodist], [AS_HELP_STRING([--with-acrodist], [use acrodist])])
+test "$with_acrodist" != no && AC_CHECK_PROGS(ACRODIST, acrodist, false)
+AC_ARG_WITH([ps2pdf], [AS_HELP_STRING([--with-ps2pdf], [use ps2pdf])])
+test "$with_ps2pdf" != no && AC_CHECK_PROGS(PS2PDF,   ps2pdf,   false)
+AC_ARG_WITH([pstopdf], [AS_HELP_STRING([--with-pstopdf], [use pstopdf])])
+test "$with_pstopdf" != no && AC_CHECK_PROGS(PSTOPDF,  pstopdf,  false)
 
 dnl Check for progs needed for manpage generation
 if test $ASCIIDOC = false; then
